modules/MicroMegaWattBSP/manifest.yaml

```diff
@@ -16,6 +16,12 @@ config:
     minimum: -1
     maximum: 1000
     default: -1
+  dc_max_voltage:
+    description: Maximum voltage to support
+    type: integer
+    minimum: 150
```

Comment:
this is smaller than the default 180V, is this intentional?

Reply:
True. I changed both to 50V, that makes more sense. Will merge now.